diff --git a/index.d.ts b/index.d.ts
index b3ab7e3d5bd85285ffe3a5bbdc30a603680d57d8..bf9a1005a3ac8116c1582b9cc4ce4042cd12f2be 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -982,13 +982,13 @@ export interface MatchPattern {
 export interface Span {
     start: number;
     end: number;
-    ctxt: number;
 }
 export interface Node {
     type: string;
 }
 export interface HasSpan {
     span: Span;
+    ctxt?: number
 }
 export interface HasDecorator {
     decorators?: Decorator[];
@@ -1457,9 +1457,9 @@ export interface Module extends Node, HasSpan, HasInterpreter {
 }
 export interface Script extends Node, HasSpan, HasInterpreter {
     type: "Script";
-    body: Statement[];
+    body: (Statement | Node)[];
 }
-export type ModuleItem = ModuleDeclaration | Statement;
+export type ModuleItem = ModuleDeclaration | Statement | Node;
 export type BinaryOperator = "==" | "!=" | "===" | "!==" | "<" | "<=" | ">" | ">=" | "<<" | ">>" | ">>>" | "+" | "-" | "*" | "/" | "%" | "|" | "^" | "&" | "||" | "&&" | "in" | "instanceof" | "**" | "??";
 export type AssignmentOperator = "=" | "+=" | "-=" | "*=" | "/=" | "%=" | "<<=" | ">>=" | ">>>=" | "|=" | "^=" | "&=" | "**=" | "&&=" | "||=" | "??=";
 export type UpdateOperator = "++" | "--";
